scripts/jenkins/cloud/ansible/roles/supportconfig/defaults/main.yml

```diff
@@ -21,4 +21,4 @@ supportconfig_pkgs:
   - supportutils-plugin-suse-openstack-cloud
 
 supportconfig_filename: "{{ cloud_env }}-{{ (inventory_hostname in groups['deployer_virt']) | ternary('admin', inventory_hostname) }}"
-supportconfig_filepath: "/var/log/nts_{{ supportconfig_filename }}.tbz"
```

Comment:
If we don't use .tbz, then we do not need to install the bzip2 package above anymore